fili-core/src/main/java/com/yahoo/bard/webservice/druid/model/filter/SearchFilter.java

```diff
@@ -1,4 +1,4 @@
-// Copyright 2016 Yahoo Inc.
+// Copyright 2017 Yahoo Inc.
```

Comment:
Looks like this file is there since 2016 (we put based on when file is initialized)